bgpd/bgp_debug.c

```diff
@@ -415,6 +418,10 @@ int bgp_dump_attr(struct attr *attr, char *buf, size_t size)
 				 inet_ntoa(attr->cluster->list[i]));
 	}
 
+	if (CHECK_FLAG(attr->flag, ATTR_FLAG_BIT(BGP_ATTR_PMSI_TUNNEL)))
+		snprintf(buf + strlen(buf), size - strlen(buf),
```

Comment:
calling strlen(buf) twice is ugly/expensive, would be better to call it once and use a variable - (this is a non blocking comment)

Reply:
Upon looking again, while this is a valid comment, this change is just following what the rest of the code in this function is doing. Since this function seems to be invoked when debug logs are turned on, I'll leave it untouched.